core/node/notifications/sync/streams_tracker.go

```diff
@@ -84,7 +84,13 @@ func (tracker *NotificationsStreamsTracker) coldStreamsEnabled() bool {
 
 // TrackStream returns true if the given streamID must be tracked for notifications.
 func (tracker *NotificationsStreamsTracker) TrackStream(streamID shared.StreamId, isInit bool) bool {
```

Comment:
There is also `AppRegistryStreamsTracker#TrackStreams`

Reply:
Done. Moved this logic to track_streams as an extra filter.